lib/sisense/api/client.rb

```diff
@@ -91,11 +91,11 @@ def handle_error(response)
         error_params = new_api_format_error_params || error_params
         case response.code
         when "404"
-          raise Sisense::API::NotFoundError, error_params
+          raise Sisense::API::NotFoundError.new(**error_params)
```

Comment:
[Positional and keyword arguments are separated in Ruby 3.0](https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/), so `raise Sisense::API::NotFoundError, error_params` fails with ArgumentError.